res/css/views/right_panel/_UserInfo.scss

```diff
@@ -290,8 +278,4 @@ limitations under the License.
         max-width: 72px;
         margin: 0 auto;
     }
-
-    .mx_UserInfo_avatar .mx_BaseAvatar_initial {
-        font-size: 40px !important; // override the other override because here the avatar is smaller
```

Comment:
surely we need to keep such a restriction and apply it to the new initial pseudo element

Reply:
![image](https://user-images.githubusercontent.com/2403652/120635975-fa4c2500-c464-11eb-9c3b-db698ab08ee3.png)

Yeah :D

Reply:
I don't think we need to apply a restriction as the `font-size` is a percentage off the `width` and `height` of the avatar.
However that shows me that the `toPx` function is not dealing great with the avatar on the right hand side

![Screen Shot 2021-06-03 at 12 21 49](https://user-images.githubusercontent.com/769871/120637179-75faa180-c466-11eb-9f9d-bf976ec3e0f6.png)

And there is a local CSS override for the width that breaks the ratio with the `font-size` explaining the gigantic letters here